app/controllers/HomeController.php

```diff
@@ -215,6 +219,51 @@ public function getSeeds(){
 
 		return $userSeeds;
 	}
+
+	public function sellItem()
+	{	
+		$price = Input::get("price_to_sell");
+		$name = Input::get("item_to_sell_name");
+		$description = Input::get('description_of_item');
+		$itemsTable = DB::table('items')->where('name' ,'=', $name)->first();		
+		$itemSold = $itemsTable->id;
+
+		$user = DB::table('users')->where('username','=',Auth::user()->username)->first();
```

Comment:
on the table of 'users' the username is unique therefore i call it like thats as more of a safecheck and it works fine so theres no reason to change it